docs/concepts/custom-ui-headless-api.mdx

```diff
@@ -0,0 +1,662 @@
+---
+id: custom-ui-headless-api
+title: Integration Basics
+sidebar_label: Integration Basics
+toc_max_heading_level: 4
+---
+
+```mdx-code-block
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+import CodeBlock from "@theme/CodeBlock"
+```
+
+Ory provides headless APIs for ease of integration. This ensures that Ory provides compatibility with many software stacks across
+different programming languages.
+
+This documents' focus is to provide concrete examples on custom UI integration. Flows are the fundamental of Ory's self-service
```

Comment:
In the end, it is not really clear how to render a form from a flow, or when to create a flow in the UI in the first place. The page does really answer the question posed here.